0001-short-term-release-process/0001-release-process-short-term.md

```diff
@@ -0,0 +1,322 @@
+# Short-Term Release Process for Autonomi
+
+- Status: proposed
+- Type: enhancement
+- Related components:
+- Start Date: 2024-07-08
+- Discussion:
+- Supersedes:
+- Superseded by:
+
+## Summary
+
+This document intends to provide a detailed development and release process we can follow until the
+initial launch of the network in October/November. It will be based on short release cycles that
+will last about three weeks. After launch, the cycles would likely be at least double that length
+and could be quite different in detail.
+
+It should be clear that we deal here with processes for releasing code, but we don't go into any
+detail on how to deploy it. There are issues there we will also have to address, but those will be
+for another document.
+
+## Conventions
+
+Uncertainties are indicated using a `*` notation. Rather than have a full `Unresolved Questions`
+section at the end, where applicable there is an `Uncertainties` list in each section.
+
+The use of `**` is to denote an idea that came from Shu's diagram/proposal.
+
+## Motivations
+
+So far, we have failed to arrive at a well-defined process for releasing our code. With launch
+imminent, it is my opinion that we've now run out of road for experimentation and we need to settle
+on an unambiguous process that we can execute without deliberation. Finalising this document should
+be a collaborative process in which we address any uncertainties rather than leaving them open for
+further experiment.
+
+## Detailed Design
+
+## Beta Release Cycles and Schedule
+
+This section provides guidelines for possible dates for each cycle until launch.
+
+### Wave 1
+
+Release and deployment for next wave:
+* Stable release and production deploy: 2024-07-08
+
+### Wave 2
+
+Live for users: 2024-07-09 / 825 people / 11 week competition / competition ends: 2024-09-27
+
+Three week development/release cycle:
+* Development begins 2024-07-08
+* Release candidate branch cut: 2024-07-22
+* Stable release and production deploy for wave 3: 2024-07-31
+
+### Wave 3
+
+Live for users: 2024-08-01 / 1000 people / 8 week competition / competition ends: 2024-09-27
+
+Three week development/release cycle:
+* Development begins 2024-08-01
+* Release candidate branch cut: 2024-08-15
+* Stable release and production deploy for wave 4: 2024-08-22
+
+### Wave 4
+
+Live for users: 2024-08-23 / 1000 people / 5 week competition / competition ends: 2024-09-27
+
+Three week development/release cycle:
+* Development begins 2024-08-22
+* Release candidate branch cut: 2024-09-05
+* Stable release and production deploy for wave 5: 2024-09-12
+
+### Wave 5
+
+Live for users: 2024-09-13 / 1000 people / 3 week competition / competition ends: 2024-09-27
+
+Three week development/release cycle:
+* Development begins 2024-09-12
+* Release candidate branch cut: 2024-09-26
```

Comment:
Seems the development/release cycle is defined as a 3 week cycle, and written as part of 5 waves.

wave 2: development begins: 07-08 -> 07-22 (14 days) then cut happens
wave 3: development begins: 08-01 -> 08-15 (14 days) then cut happens
wave 4: development begins: 08-22 -> 09-05 (14 days) then cut happens
wave 5: development begins: 09-12 -> 09-26 (14 days) then cut happens

Am I misinterpreting this because development should always be ongoing on main, the way its written its as if development stops between 07-22 and 08-01 and so forth on the gaps between the 4 waves above.